easybuild/easyconfigs/c/COSMO_dycore/COSMO_dycore-1596740a01fff0c3ab036fda09da49d137830b62-CrayGNU-17.08-float.eb

```diff
@@ -0,0 +1,35 @@
+# jg (cscs)
+easyblock = 'CMakeMake'
+
+name = 'COSMO_dycore'
+version = '1596740a01fff0c3ab036fda09da49d137830b62'
+versionsuffix = '-float'
+
+homepage = 'https://github.com/MeteoSwiss-APN/cosmo-pompa'
+description = """cosmo-pompa"""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'verbose': False}
+
+# can't clone private repo github.com/MeteoSwiss-APN/cosmo-pompa.git (+ no official release)
```

Comment:
Shouldn't it be the EB-equivivalent of: 
```
git clone -b crclim git@github.com:C2SM-RCM/cosmo-pompa.git
```
respectively
```
git clone -b crclim https://github.com/C2SM-RCM/cosmo-pompa
```

Reply:
jenscscs has no access to private repo

Reply:
But we can store the version of the crclim branch in /apps/common/UES/easybuild/sources. You don't want that?